.stylelintrc

```diff
@@ -2,7 +2,7 @@
   "extends": "stylelint-config-standard",
   "rules": {
     "no-empty-source": null,
-    "selector-class-pattern": "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
+    "selector-class-pattern": "^([a-z][a-z0-9]*)((-|_)[a-z0-9]+)*$",
```

Comment:
update this pattern due to we have a new css selector `.section-item-content > .awsui_input-container_2rhyz_dvvio_218`